public/pages/CreateTrigger/containers/DefineBucketLevelTrigger/DefineBucketLevelTrigger.js

```diff
@@ -309,7 +309,12 @@ class DefineBucketLevelTrigger extends Component {
         <div style={{ padding: '0px 10px', paddingTop: '10px' }}>
           <FormikFieldText
             name={`${fieldPath}name`}
-            fieldProps={{ validate: validateTriggerName(triggers, triggerValues, fieldPath) }}
+            fieldProps={{
+              validate: (val) => {
+                var r = validateTriggerName(triggerValues?.triggerDefinitions, triggerIndex)(val);
```

Comment:
Is `(val)` a valid syntax? What is it passing over?

Reply:
you could treat it a func of func, the validate is a func to deal with a value returned from validateTriggerName (our customized validation)